include/irr/asset/ISampler.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __IRR_I_SAMPLER_H_INCLUDED__
+#define __IRR_I_SAMPLER_H_INCLUDED__
+
+#include "irr/asset/SSamplerParams.h"
+
+namespace irr {
+namespace asset
+{
+
+class ISampler
+{
+protected:
+    ISampler(const SSamplerParams& _params) : m_params(_params) {}
+    virtual ~ISampler() = default;
+
+    SSamplerParams m_params;
```

Comment:
add `SSsamplerParams` inline inside `ISampler` (could be `ISampler::Params`)

Reply:
resolved